src/components/SearchBar/SearchBar.tsx

```diff
@@ -24,9 +24,10 @@ const SearchInput = styled.input`
   display: flex;
   border: ${palette.purple} solid 1px;
   width: 100%;
-  ${basicSourceSansText};
+  font-family: 'Source Sans Pro';
+  font-style: normal;
```

Comment:
Here too, like if we have to define Source Sans Pro in a lot of files, then when we change the font-familky - its a chore again.
We could argue thought that the name basicSourceSansText is not ideal. Maybe **baseFont** would be better

Reply:
Will be fixed soon, don't worry!